notification-service/build.gradle

```diff
@@ -21,14 +21,15 @@ configurations {
 }
 
 repositories {
```

Comment:
**[THIS_IS_FOR_A_FEW_LINES_ABOVE]** ([line 17-19](https://github.com/dmitrynovik/rabbitmq-transaction-processing/pull/2/files#diff-4d35d8cec5be62506f1a64ac9dd276d4a6c7b6891db6aae96583408938feef04R17-R19))
Out of curiosity why are you having to do this?